cmd/remoteenforcer/remoteenforcer_linux.go

```diff
@@ -44,7 +44,7 @@ func NewServer(service enforcer.PacketProcessor, rpchdl rpcwrapper.RPCServer, rp
 		if err != nil {
 			return nil, err
 		}
-		procMountPoint := os.Getenv(envProcMountPoint)
+		procMountPoint := os.Getenv(constants.AporetoEnvMountPoint)
 		if len(procMountPoint) == 0 {
```

Comment:
it's better to check for empty string. It makes it clearer that this guy returns a string